pkg/tcpip/header/icmpv6.go

```diff
@@ -54,6 +54,15 @@ const (
 	// address.
 	ICMPv6NeighborAdvertSize = ICMPv6HeaderSize + NDPNAMinimumSize + NDPLinkLayerAddressSize
 
+	// ICMPv6RouterAdvertMinimumSize is the minimum size of a
+	// router advertisement packet.
+	ICMPv6RouterAdvertMinimumSize = ICMPv6HeaderSize + NDPRAMinimumSize
+
+	// ICMPv6RouterAdvertSize is size of a router advertisement
```

Comment:
I don't think this is needed. See comments below about an RA containing 0 or more prefix information options, and the source link layer address option being conditional